amundsen_application/static/js/components/common/Table/testDataBuilder.ts

```diff
@@ -0,0 +1,74 @@
+const defaultData = [
```

Comment:
Something I don't runderstand about this yet is that it appears to be a helper method for the test `setup()` where propoverrides + jest methods like `beforeAll()` can be used to set up different scenarios. 

Any examples you want to merge in will help with discussion though, which would be centered around understanding the value of creating another file/layer of code and why its abstracted out in this way (over let's say, being built into the `setup()` method support.

Reply:
Yep, it can be used inside the setup method, but also within each test case to pass the values into the setup method.

The utility of this extra layer is the flexibility it gives us, as we can use it on unit tests as we do here, and also on stories for our Storybook and for End to End tests with Cypress in the future.